gateway-api-router/Makefile

```diff
@@ -0,0 +1,59 @@
+# Run controllers tests with two nodes by default to (potentially) minimise
+# flakes.
+CONTROLLERS_GINKGO_NODES ?= 2
+ifdef GINKGO_NODES
+CONTROLLERS_GINKGO_NODES = $(GINKGO_NODES)
+endif
+
+# Setting SHELL to bash allows bash commands to be executed by recipes.
+# This is a requirement for 'setup-envtest.sh' in the test target.
+# Options are set to exit when a recipe line exits non-zero or a piped command fails.
+SHELL = /usr/bin/env bash -o pipefail
+.SHELLFLAGS = -ec
+
+##@ General
+
+# The help target prints out all targets with their descriptions organized
+# beneath their categories. The categories are represented by '##@' and the
+# target descriptions by '##'. The awk commands is responsible for reading the
+# entire set of makefiles included in this invocation, looking for lines of the
+# file as xyz: ## something, and then pretty-format the target and help. Then,
+# if there's a line with ##@ something, that gets pretty-printed as a category.
+# More info on the usage of ANSI control characters for terminal formatting:
+# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
+# More info on the awk command:
+# http://linuxcommand.org/lc3_adv_awk.php
+
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Development
+export GOBIN = $(shell pwd)/bin
+export PATH := $(shell pwd)/bin:$(PATH)
+
+webhooks-file = ../helm/korifi/gateway-api-router/manifests.yaml
```

Comment:
After we move all webhooks back to the core controllers we will not need the webhook manifest generation infrastructure. These webhooks are right now duplicated anyway.